sdk/keyvault/Azure.Security.KeyVault.Keys/CHANGELOG.md

```diff
@@ -58,6 +61,13 @@
 
 - Default service version is now 7.1.
 
+## 4.1.0-preview.1 (2020-03-09)
```

Comment:
This doesn't look correct.

Reply:
You are hitting https://github.com/Azure/azure-sdk-tools/issues/1389. @chidozieononiwu made a change recently that sorts these by version but I think we will switch to sorting by date.

In the mean time someone could push an update to revert this part in this PR or create another PR.

Reply:
That's what I'm going to do since it also re-sorts by version (which, IIRC, has been the case since we started doing this auto-PR).